src/controllers/api/monsterController.js

```diff
@@ -10,14 +13,23 @@ exports.index = async (req, res, next) => {
     search_queries.challenge_rating = { $in: req.query.challenge_rating.split(',') };
   }
 
-  await Monster.find(search_queries)
-    .sort({ index: 'asc' })
-    .then(data => {
-      res.status(200).json(utility.NamedAPIResource(data));
-    })
-    .catch(err => {
-      next(err);
-    });
+  const redisKey = req.originalUrl;
+  const data = await getAsync(redisKey);
```

Comment:
Might be worth having some error handling for this in case of redis error.

Reply:
Good idea!